CharacterCreation/Class-Specific Documentation/Doctor/Doctor Hippocratic Proceedures.txt

```diff
@@ -0,0 +1,337 @@
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
+
+==Defib ==
+Adrenaline Reward:  80 Points
+Contagion Reward: 2 Contagion
+Range: Touch
+Action: 2 Actions
+
+    You adjust the dials on your biofoam injector to compensate for loss of
+vital fluids and to administer the correct frequency and wattage of tachycardia
+interuptor shock. This is gonna hurt like hell, but they'll live to complain 
+about it at least. Who knows? Maybe they'll even say "Thank you!" this time?
+
+    You must have your Field Applicator drawn in order to use this ability. Only 
+usable on an organic patient that is bleeding out. The person is no longer 
+downed and has 10+2d10 Health Points, rather than having to stabilize the target 
+and then heal them as a separate action.
+
+== Pre-op traige ==
+Adrenaline Reward:  30 Points
+Contagion Reward: 1 Contagion
+Range: 10m
+Action: 1 Action
+
+    You scan the patient and dial your Field Applicator to match the patient's
+unique biochemestry. You also take the time to more closely inspect the patient's
+injuries and prognosticate the battlefield's effect on their condition. Your
+keen eye and experience allows you to better treat your patient's maladies. 
+
+    You must have your Field Applicator drawn in order to use this ability. This 
+trick only works on biologic patients. If you use another procedure on this 
+patient either this turn or the next turn that says you can only heal them up
+to half health, you may ignore that limitation.
+
+==Remote Surgery ==
+Adrenaline Reward:  20
+Range:          20m
+Action:         2 Actions
+
+    Let's take this show on the road! You carefully feed out an omni-glob from
+your trusty applicator and throw it at your distant patient. Then, turning to
+your applicator, direct its functions from afar. 
+
+    You must have your Field Applicator drawn in order to use this ability, and 
+you must be able to see your patient. Your patient heals an ammount of health 
+listed on the table below. As your character learns more medicine skill, the 
+better they heal. Unless otherwise stated, this ability cannot heal a patient 
+above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       14        |     10       | + 2d10 |
++-----------------+--------------+--------+
+|       24        |     20       | + 3d10 |
++-----------------+--------------+--------+
+|       34        |     25       | + 4d10 |
++-----------------+--------------+--------+
+|       44        |     30       | + 5d10 |
++-----------------+--------------+--------+
+|       54        |     40       | + 6d10 |
++-----------------+--------------+--------+
```

Comment:
Any updates to the table on patch-em-up, if any, would have to cascade down here too.